pkg/controller/migplan/migplan_controller.go

```diff
@@ -218,6 +220,13 @@ func (r *ReconcileMigPlan) Reconcile(request reconcile.Request) (reconcile.Resul
 		return reconcile.Result{Requeue: true}, nil
 	}
 
+	// Set excluded resources on Status.
+	err = r.ensureExcludedResourceList(plan)
```

Comment:
After seeing it implemented, I'm not convinced that _ensure_ is the best fit.  Seems like it may imply more than it's doing since we typically use _ensure_ when affecting the outside world.  I don't feel strongly.

Perhaps something like `setExcludedResources()`.

Reply:
I can use set. Keep in mind that eventually this set method may be merging the EXCLUDE_RESOURCES env var and a future Spec.ExcludedResources slice. If set still works for that, then I can change to set here.